cli/src/dev-server/plane.ts

```diff
@@ -97,9 +97,8 @@ export class LocalPlane {
     const stdout = readline.createInterface({ input: logsProcess.stdout }).on('line', callback)
 
     const errorLogs: string[] = []
-    const stderr = readline.createInterface({ input: logsProcess.stderr }).on('line', (line: string) => {
-      errorLogs.push(line)
-    })
+    console.log(errorLogs)
```

Comment:
We can remove this `console.log` statement. Also, looks like we don't use `errorLogs` anymore, so we can drop that too

Reply:
ohh yes thanks!